docs/lsu.md

```diff
@@ -0,0 +1,85 @@
+# LSU
+The design of the LSU is losely based on the LSU implementation in [BOOM](https://docs.boom-core.org/en/latest/sections/load-store-unit.html) microarchitecture and the [E500](https://www.nxp.com/docs/en/reference-manual/E500CORERM.pdf) core.
+
+### Ports
+in_lsu_insts <--> dispatch (Receive new memory instructions)
+
```

Comment:
It would be clarify the direction (w.r.t. LSU) explicitly, such as:
* in_lsu_insts:  input from Dispatch; receive new memory instructions
 * in_cache_loop_ack:  input from DCache; receive acknowledgement from cache....